app/services/cache_service.rb

```diff
@@ -0,0 +1,51 @@
+# service class for pre-caching salesforce listings
+class CacheService
```

Comment:
@creativecoder would be interested to discuss `CacheService` with you and get your thoughts. As I mentioned last week we had to make some decisions/compromises as how to best pre-cache listings on a regular basis, and how frequently to do it. I think it's in a "good enough to deliver" place but I think it's worth one more conversation to finalize.